src/components/Calendar/Calendar.js

```diff
@@ -164,6 +158,11 @@ const MyCalendar = (props) => {
     }
   };
 
+  // useEffect for select location dropdown
+  useEffect(() => {
```

Comment:
I know we said, we would leave this in for Carlos to build off of but maybe we should comment it out for now? Just so it's not being called unnecessarily before it is built.

Reply:
Thats a great point. I'll comment it out and push the code again

Reply:
I commented that useEffect out for Carlos to use in the future and re-pushed my code. Thank you for catching that Pete.

Reply:
Thanks for doing that!

Reply:
No problem Carlos